sdk/instrumentation/bodyattribute/bodyattribute.go

```diff
@@ -69,3 +75,22 @@ func SetEncodedBodyAttribute(attrName string, body []byte, truncated bool, span
 		span.SetAttribute(fmt.Sprintf("%s.truncated", attrName), true)
 	}
 }
+
+func truncateUTF8Bytes(b []byte, maxBytes int) []byte {
```

Comment:
Maybe a comment pointing to original code is helpful in case we want to look for updates or bugfixes.